features/dynamic-bundle-discounts.md

```diff
@@ -659,4 +643,53 @@ if nostoBundleHandler.isValid()
 end
 
 Output.cart = Input.cart
-```
\ No newline at end of file
+```
+
+## Troubleshooting
+
+### Issue #1: When additional content displayed in cart page
+
+![Properties Exposed](https://user-images.githubusercontent.com/82023195/168611801-666c24bc-c7f9-4edd-bdf5-2be32798391c.png)
+
+### Cause
+
+Nosto implementation uses a private property field for sharing discount information with Shopify. In newer theme versions, Shopify automatically hides these fields while displaying products in cart page. In older version of themes, these fields gets exposed as shown below.
```

Comment:
> private property field
Do you mean a hidden form value? e.g `<input type="hidden">`?

Reply:
Rather it's something that's not supposed to be exposed. "Technically" we can assume it's a hidden field